src/test/java/com/tyche/ramsees/strategies/MacDBasedStrategyMock.java

```diff
@@ -25,22 +23,23 @@ public MacDBasedStrategyMock(BarSeries series) {
     }
 
     @Override
-    public void logStatus(Bar lastBae, Integer endIndex) {
+    public void logStatus(Bar lastBar, Integer endIndex) {
         if (strategy == null) {
             return;
         }
 
-        BarSeriesManager seriesManager = new BarSeriesManager(series, new LinearTransactionCostModel(0.1), new ZeroCostModel());
-        // We start by the index number TREND_EMA to have enough data to calculate EMAs
+        BarSeriesManager seriesManager = new BarSeriesManager(series, new LinearTransactionCostModel(0.001), new ZeroCostModel());
```

Comment:
feePerPosition – the feePerPosition coefficient (e.g. 0.001 for 0.1% per trade)

0.1 was adding a fee of 10% 😄 that's why we had 0 winning positions!